holidays/countries/fiji.py

```diff
@@ -0,0 +1,182 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.calendars import _CustomHinduHolidays, _CustomIslamicHolidays
+from holidays.calendars.gregorian import SEP, OCT, NOV
+from holidays.constants import PUBLIC, WORKDAY
+from holidays.groups import (
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_SUN_TO_NEXT_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+    ALL_TO_NEAREST_MON,
+)
+
+
+class Fiji(
+    ObservedHolidayBase,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    ChristianHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """
+    References:
+    - https://laws.gov.fj/Acts/DisplayAct/2910#
+    - https://laws.gov.fj/LawsAsMade
+    - https://www.fiji.gov.fj/About-Fiji/Public-Holidays
+    - https://www.timeanddate.com/holidays/fiji/
+    - https://en.wikipedia.org/wiki/List_of_festivals_in_Fiji
+    - https://www.rnz.co.nz/international/pacific-news/249514/new-public-holiday-for-fiji
+    - https://www.fijitimes.com.fj/constitution-day-public-holiday-removed-cabinet/
+    - https://fijivillage.com/news/National-Sports-Day-celebrated-5krs29/
+    - https://fijivillage.com/news/Cabinet-approves-Ratu-Sir-Lala-Sukuna-Day-and-Girmit-Day-and-removes-Constitution-Day-as-a-public-holiday-f48r5x/
+    """
+
+    country = "FJ"
+    supported_categories = (PUBLIC, WORKDAY)
+    # %s (estimated).
+    estimated_label = "%s (estimated)"
+    # %s (observed).
+    observed_label = "%s (observed)"
+    # %s (observed, estimated).
+    observed_estimated_label = "%s (observed, estimated)"
+    # Act No. 13 of 2015
+    start_year = 2016
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=FijiHinduHolidays, show_estimated=True)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self, cls=FijiIslamicHolidays, show_estimated=True)
+        StaticHolidays.__init__(self, FijiStaticHolidays)
```

Comment:
Now we don't need `FijiStaticHolidays` (and `StaticHolidays` too). Sources from "Official Fiji Public Holidays Calendar" can be moved to Fiji docstring.